lib/imapoptions

```diff
@@ -1293,6 +1293,12 @@ Blank lines and lines beginning with ``#'' are ignored.
    For backward compatibility, if no unit is specified, kibibytes is assumed.
    */
 
+{ "jmap_max_calendareventnotifs", 200, INT, "UNRELEASED" }
+/* The maximum count of CalendarEventNotification objects to keep per account.
+   Any notifications exceeding this count are deleted to make room for new
+   ones. Zero or any negative number disables this limit.
+    */
+
```

Comment:
```suggestion
{ "jmap_max_calendareventnotifs", 200, INT, "UNRELEASED" }
/* The maximum count of CalendarEventNotification objects to keep per account.
   Any notifications exceeding this count are expunged to make room for new
   ones. Zero or any negative number disables this limit.
    */

```

I think it's better to say "expunged" rather than "deleted" here to be technically correct about what happens, since "deleted" is kind of a loaded word wrt email.

If I understand this change correctly, all we're doing here is marking the messages as expunged, and therefore in most cases cyr_expire will still need to run at some point for them to really disappear?  I don't think it's necessary to document that detail here, as long as we say "expunged", I think we can trust that the admin remembers (or can look up) what `expunge_mode` they've configured and what ramifications it has...

Reply:
done